com.chartboost.helium/Editor/HeliumIntegrationChecker.cs

```diff
@@ -0,0 +1,329 @@
+            }
+
+            ReimportExistingHeliumSamples(importedDependencies, helium.version);
+            return true;
+        }
+
+        /// <summary>
+        /// Uncomment UnityAds dependency on Optional-HeliumUnityAdsDependencies.xml if present.
+        /// </summary>
+        /// <param name="skipDialog">Optional parameter to skip any dialog windows, if true, UnityAds dependency will be uncommented if possible.</param>
+        public static void UncommentUnityAdsDependency(bool skipDialog = false)
+        {
+            var helium = FindPackage(HeliumPackageName);
+            if (!Version.TryParse(helium.version, out var heliumFoundVersion))
+            {
+                if (!skipDialog)
+                {
+                    EditorUtility.DisplayDialog(
+                        HeliumWindowTitle,
+                        $"Failed to parse version {helium.version} in Package.\n\n**This is probably a bad setup, contact Helium Support at support@chartboost.com**",
+                        "Ok");
+                }
+                return;
+            }
+
+            var heliumVersion = heliumFoundVersion.ToString();
+            var unityAdsDependencyPath = $"Assets/Samples/Helium SDK/{heliumVersion}/UnityAds/Editor/Optional-HeliumUnityAdsDependencies.xml";
+
+            // check if UnityAds is integrated
+            if (!File.Exists(unityAdsDependencyPath))
+                return;
+
+            var unityAdsDependencyLines = File.ReadLines(unityAdsDependencyPath).ToList();
+            var commentedLineIndex = unityAdsDependencyLines.FindIndex(line => line.Contains(UnityAdsSDKCommented));
+            if (commentedLineIndex == -1)
+                return;
+
+            var updateUnityAdsSample = true;
+            if (!skipDialog)
+            {
+                updateUnityAdsSample = EditorUtility.DisplayDialog(
+                    HeliumWindowTitle,
+                    "Helium UnityAds Ad Adapter found, but UnityAds dependency is commented. This will lead to a non-functional adapter.\n\nDo you wish to uncomment it?",
+                    "Yes", "No", DialogOptOutDecisionType.ForThisMachine, UnityAdsUncommentWindow);
+            }
+            if (!updateUnityAdsSample)
+                return;
+            unityAdsDependencyLines[commentedLineIndex] = UnityAdsSDKUncommented;
+            File.WriteAllLines(unityAdsDependencyPath, unityAdsDependencyLines);
+        }
+
+        [MenuItem("Helium/Integration/UnityAds Check", false, 1)]
+        public static void CheckUnityAdsIntegration()
+        {
+            var helium = FindPackage(HeliumPackageName);
+            if (!Version.TryParse(helium.version, out var heliumFoundVersion))
+            {
+                EditorUtility.DisplayDialog(
+                    HeliumWindowTitle, 
+                    $"Failed to parse version {helium.version} in Package.\n\n**This is probably a bad setup, contact Helium Support at support@chartboost.com**",
```

Comment:
Perhaps, we should send pubs/users to the docs first, before contacting support.

Reply:
@cb-jpadilla This should probably never happen, as our versioning is very consistent. The docs do not contain any information on how/why this error is produced, so if this ever happen it would be good for it to come back to us.